src/js/other-websites/add_input_icon.js

```diff
@@ -209,9 +234,15 @@ browser.runtime.onMessage.addListener(function(m, sender, sendResponse) {
   }
 
   // This event is fired from the iframe when the user presses "Escape" key or completes an action (Generate alias, manage aliases)
-  if (m = "iframeCloseRelayInPageMenu") {
+  if (m == "iframeCloseRelayInPageMenu") {
```

Comment:
This looks like a bug that ESLint would've caught. I'll look at setting that up for this repo too.